terraform-modules/storage-bucket/bucket.tf

```diff
@@ -11,6 +11,11 @@ resource "google_storage_bucket" "bucket" {
     enabled = var.versioning
   }
 
+  logging {
```

Comment:
will this work if log_bucket and prefix is not passed in or needed?

Reply:
You might want to use a dynamic block similar to 

  dynamic "retention_policy" {
    for_each = var.retention_policy == null ? [] : [var.retention_policy]
    content {
      is_locked        = var.retention_policy.is_locked
      retention_period = var.retention_policy.retention_period
    }
  }

Granted you can only have a single value so for_each is kind of odd.  but I am not sure how else to only include a section if var is set?  there may be a better or another way?

Reply:
Great idea - I did a quick search but didn't see anything better than the way you suggested.